### PR TITLE
Performance check collinearity

### PR DESCRIPTION
**Pull request by DominiqueMakowski**

### CODE REVIEW

**Comment by DominiqueMakowski** [comment]
Let me know whenever you want to me to build and merge it ☺️

**Comment by strengejacke** [comment]
I started the blog post and realized there was a "bug" in the plot-method for models with zero-inflation. So I improved the plot-method, fixed an issue in performance, and thought maybe it makes more sense to publish this post once the package updates are on CRAN. So I maybe start with another post here... ;-)

**Comment by DominiqueMakowski** [comment]
Sure! Maybe for the release of parameters 😬

**Comment by strengejacke** [comment]
ok, you may publish this (after a short cross-check, if you want).

**Comment by DominiqueMakowski** [comment]
I built the website. Can merge whenever you wan (or maybe do wait like 12h so I'll be awake if there are any issues again with paths and stuff (it's been a while since we haven't posted anything ^^))

**Comment by strengejacke** [comment]
I just saw that building the site replaced all absolute by relative URLs (`https://easystats.github.io/blog/` => `blog/`) - is that intended?

**Comment by strengejacke** [comment]
Quoted from the readme:

> Once you're satisfied, close R and R studio. Go to /docs/categories/r and open index.xml with a notepad. Search for all the /blog/blog/ strings and replace by https://easystats.github.io/blog/ if you see that the URLs is relative or just by blog otherwise.

File: docs/categories/performance/index.html [hunk 1]
```diff
@@ -54,12 +54,12 @@
...
     <!-- Icons -->
-    <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/apple-touch-icon-144-precomposed.png">
-    <link rel="shortcut icon" href="/favicon.png">
+    <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/blog/apple-touch-icon-144-precomposed.png">
+    <link rel="shortcut icon" href="/blog/favicon.png">
     
 
-<link href="https://easystats.github.io/blog/categories/performance/index.xml" rel="alternate" type="application/rss+xml" title="easystats" />
-<link href="https://easystats.github.io/blog/categories/performance/index.xml" rel="feed" type="application/rss+xml" title="easystats" />
+<link href="/blog/blog/categories/performance/index.xml" rel="alternate" type="application/rss+xml" title="easystats" />
```
Comment (strengejacke) [inline docs/categories/performance/index.html:61]: Here we now have these `blog/blog/` paths.

**Comment by DominiqueMakowski** [comment]
TBH I don't really understand how this works, but I *think* that there are two components, one is the blog on its own website, and there is the RSS feed for rbloggers. The RSS feed to which rbloggers is connected to is in `/docs/categories/r`, in which I have replaced the relative URLs in theory

as for the blog at the blog address, it should work as it worked before

**Comment by DominiqueMakowski** [comment]
But I haven't managed to get a general or abstract understanding of how this stuff works, this pipeline came out of trials and errors, so every time we hit merge I'm like fingers crossed ^^

**Comment by strengejacke** [comment]
OMG all images broken.... I try to fix this.

**Comment by strengejacke** [comment]
Ok, literally every URL (images?) is broken now... :-/ I fixed some files, in hope that the blog will work, but I'm not sure which strings in which files have to be replaced...

**Pull request merged**